### PR TITLE
chore: test case for command suggestion feature

### PR DESCRIPTION
**Pull request by jamesgeorge007**
Summary:
---------
Follow up of #1001 

Test Plan:
----------
Covers the test case for #960

### CODE REVIEW

**Comment by jamesgeorge007** [comment]
@thymikee having an additional `lint:fix` script would be a good idea.

**Comment by thymikee** [comment]
I don't think so. You can run `yarn lint --fix`

**Comment by jamesgeorge007** [comment]
@thymikee hope we're good to land :rocket:

File: __e2e__/validations.test.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import {run, getTempDirectory, cleanup, writeFiles} from '../jest/helpers';
+
+const DIR = getTempDirectory('command-validations');
```
Comment (thymikee) [inline __e2e__/validations.test.ts]: Can you rename this to `test_didyoumean` and the file to `didYouMean.test.ts`?

**Pull request merged**